pkg/cloud/volume_limits.go

```diff
@@ -28,6 +28,7 @@ const (
 	nitroMaxAttachments                  = 28
 )
 
+//nolint:gochecknoinits // TODO Refactor to avoid using init function to prevent side-effects
```

Comment:
Unless its already in the backlog, could we create an issue about this so as to not forget it?